example/reinforcement-learning/a3c/rl_data.py

```diff
@@ -21,7 +21,7 @@
 import gym
 import cv2
 import math
-import Queue
+import queue
```

Comment:
>>> import queue
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named queue

Reply:
I see your point. Strangely, this works for me in the DLAMI with both mxnet_p27 and mxnet_p36 conda environments.
I will make the change update this PR so it works outside the DLAMI too.

Reply:
Any updates??

Reply:
The fix is simple but I have not been able to reproduce the error in order to validate the fix. 
I will anyway update the PR tomorrow.